src/app/app.constants.json

```diff
@@ -1,9 +1,9 @@
 {
-  "appname": "Angular Admin",
+  "appname": "Angular Seed",
   "scope": "FullAccess",
-  "clientid": "XXXXXXXX-XXXX-XXXX-XXXXXXXXXXXX",
-  "buyerid": "XXXX",
-  "catalogid": "XXXX",
+  "clientid": "D89E2196-CC75-4312-A4A9-67E4CC4BEDFB",
+  "buyerid": "A",
+  "catalogid": "SHARED",
   "environment": "prod",
   "defaultstate": "home",
```

Comment:
Can you remove you clear out your constants. Since it is public, we won't want to commit any of our client ids or things like that.